loco-cli/Cargo.toml

```diff
@@ -48,6 +47,7 @@ cfg-if = "1.0.0"
 dunce = { version = "1.0.4" }
 strum = { version = "0.26", features = ["derive"] }
 colored = "2"
+unicode-xid = "0.2.6"
```

Comment:
i think we can drop checking for unicode by the fact that we don't allow unicode in app name.
its reasonable to just say app names are alphanumeric (ASCII)